python/dgl/nn/pytorch/conv/graphconv.py

```diff
@@ -25,6 +155,20 @@ class GraphConv(nn.Module):
     (i.e.,  :math:`c_{ij} = \sqrt{|\mathcal{N}(i)|}\sqrt{|\mathcal{N}(j)|}`),
     and :math:`\sigma` is an activation function.
 
+    If a weight tensor on each edge is provided, the weighted graph convolution is defined as:
+
+    .. math::
+      h_i^{(l+1)} = \sigma(b^{(l)} + \sum_{j\in\mathcal{N}(i)}\frac{e_{ji}}{c_{ij}}h_j^{(l)}W^{(l)})
+
+    where :math:`e_{ji}` is the scalar weight on the edge from node :math:`j` to node :math:`i`.
+    This is NOT equivalent to the weighted graph convolutional network formulation in the paper.
+    For scalar edge weight, users may call ``EdgeWeightNorm`` on the weight tensor, then pass it
+    to ``GraphConv`` with ``norm='none'``.
+
+    To customize the normalization term :math:`c_{ij}`, one can first set ``norm='none'`` for
+    the model, and send the pre-normalize :math:`e_{ji}` to the forward computation. Please make
+    sure that `e_{ji}` is broadcastable with `h_j^{l}`.
```

Comment:
With "broadcastable", does this module support multidimensional edge weights?

Reply:
We don't disable that. Users need to take care of the normalization though.